### DISCUSSION TITLE
[M-Z] (2024/25) Foglio 6 - Esercizio 16

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#578] [Esercizi M-Z]
![es16](https://github.com/user-attachments/assets/b160dd96-0320-4a68-9ce8-fc61276d835e)

**Comment by alem1105** [comment]
![image](https://github.com/user-attachments/assets/108b2269-662c-44f1-9611-42c4425e0cb1)
